### PR TITLE
Since I know admins love popups, here's a lot of confirmation messages.

### LINKED ISSUE #5189: (WEB REPORT BY: Oninnimouse REMOTE: ) Being able to cancel out of custom virus creation when?
But in all seriousness, a lot of the things in the secret tab lack a Cancel button in case you have second thoughts, custom virus creation being one of them.

### PR DESCRIPTION
**Pull request by Intigracy**
Also removes hrefs that are no longer used anymore from the secrets panel.

fixes #5189

If there's a duplicate (as in two messages popping up in a row), you're free to complain afterwards and I'll fix it. I'm not going to go through and test every verb and href.

### CODE REVIEW

**Comment by Probe1** [comment]
Goddamn fuck off with the pop ups I don't need to pick a deity every time I SM someone

I can't even see the changes because githubs lagging

**Comment by Intigracy** [comment]
This is confirmation for all the secret panel shit.

Like explode all bombs.

Or trigger meteor storm.

**Comment by Probe1** [comment]
I don't think anyone has accidentally pressed a secrets button ever but fine.

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -917,6 +912,9 @@ var/global/floorIsLava = 0
 	set desc="Globally Toggles OOC"
 	set name="Toggle OOC"
 
+	var/confirm = alert(src, "[ooc_allowed ? "Do you hate fun and want to turn off OOC?" : "Are you a wonderful person and want to re-enable OOC?"]", "Confirm", "Yes", "No")
```
Comment (ghost) [inline code/modules/admin/admin.dm:915]: No.

Comment (SkowronX) [inline code/modules/admin/admin.dm:915]: > meme code

Comment (clusterfack) [inline code/modules/admin/admin.dm]: I warned you bro, I warned you about the memes bro

Comment (Intigracy) [inline code/modules/admin/admin.dm:915]: Only Satan turns off OOC.

Also you don't even log in anymore celt, not an issue :^).

Comment (Probe1) [inline code/modules/admin/admin.dm:915]: I really don't need a confirmation for this.

**Comment by clusterfack** [comment]
Yeah like fucking 90% of these aren't necessary

**Comment by Intigracy** [comment]
¯\_(ツ)_/¯

**Pull request closed without merging**